go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -134,9 +134,9 @@ type throttlerTopoService interface {
 // Throttler is the main entity in the throttling mechanism. This service runs, probes, collects data,
 // aggregates, reads inventory, provides information, etc.
 type Throttler struct {
-	keyspace string
-	shard    string
-	cell     string
+	keyspace    string
+	shard       string
+	tabletAlias *topodatapb.TabletAlias
```

Comment:
This change (and everything in this file) is a small refactor. It is not strictly required for this PR, but will be needed in part 2, so I thought I may as well do it now.